extensions/hawkbit-extension-artifact-repository-s3/src/main/java/org/eclipse/hawkbit/artifact/repository/S3RepositoryProperties.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.artifact.repository;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * The AWS S3 configuration properties for the S3 artifact repository
+ * implementation.
+ */
+@ConfigurationProperties("org.eclipse.hawkbit.repository.s3")
+public class S3RepositoryProperties {
+
+    private String bucketName = "artifactrepository";
+    private boolean serverSideEncryption = false;
+    private String serverSideEncryptionAlgorithm = "AES256";
```

Comment:
The S3 client has to support the algorithm. So better go for `SSEAlgorithm.AES256.getAlgorithm()`. Than it is clear what options do you have